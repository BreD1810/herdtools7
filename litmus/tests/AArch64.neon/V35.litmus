AArch64 V35
(* Test Neon final state checking
   LD1 single structure *)
{
x = 22;
0:X0=x;
}
 P0                     ;
 LD1 {V0.S}[3],[X0]     ;
forall(0:V0.S = {0,0,0,22})
