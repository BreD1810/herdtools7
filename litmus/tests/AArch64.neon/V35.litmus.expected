Test V35 Required
Histogram (1 states)
1000000:>0:V0.S={0,0,0,22};
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (0:V0.S={0,0,0,22}) is validated
Hash=441fc810b164031af66e1c561765d4d1
Observation V35 Always 1000000 0
